src/ui/theme/designSystemTypographie.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import { ComponentStory } from '@storybook/react'
+import React from 'react'
+
+import { Spacer } from 'ui/components/spacer/Spacer'
+import { TypoDS } from 'ui/theme/designSystemTypographie'
+
+export default {
+  title: 'Fondations/TypoDS',
+}
+
+export const Typographies: ComponentStory<typeof TypoDS.Title1> = () => (
+  <React.Fragment>
+    <Spacer.Column numberOfSpaces={2} />
+    <TypoDS.Title1>Title1</TypoDS.Title1>
```

Comment:
Une répétition :)
Comment faire une boucle ?

Reply:
Storybook est un espace de documentation de comment utiliser le composant dans un cas nominal

Sur Storybook, il y a un bouton "Show code" https://master--61fd537ecf081f003a135235.chromatic.com/?path=/docs/fondations-typo--typographies qui permet de voir un exemple concret de comment utiliser le composant

Faire une boucle enlèverait cette répétition mais utiliserait les composants d'une manière peu habituelle

IHMO, ça nuirait à l'auto-documentation que propose Storybook

Reply:
Ok pour le cas général et mettre 3 ou 4 exemples différents. Je comprends et je te rejoins.
Pour le côté Kitchen par contre, une petite boucle ne serait-elle pas mieux ?

Reply:
Kitchen ? de quoi parles-tu ?

Reply:
Les kitchens de composants : un endroit où tu as tous les composants. Un peu comme notre debug + storybook actuel.

This is just a proposal. Pas besoin de perdre du temps sur ce point précis.